charts/consul/test/unit/terminating-gateways-deployment.bats

```diff
@@ -1175,161 +865,18 @@ key2: value2' \
   [ "${actual}" = "value2" ]
 }
 
-#--------------------------------------------------------------------
-# terminating-gateway-init init container command
-
-@test "terminatingGateways/Deployment: terminating-gateway-init init container defaults" {
```

Comment:
These should not be deleted because the init container still exists but there is now different logic.

Reply:
Gotcha, I replicated all of the coverage in with new tests that are a little different. The gateway namespace ones are moved to a section called `consul namespaces` which starts on line 965 because they now look for annotations rather than having the value passed in to connect-init.